README.md

```diff
@@ -185,39 +186,161 @@ Additional prerequisites are provided in the different sections.
+
+While the requirement for the persistence is to be compliant to the NGSI-LD API v1.6, the currently recommended Context-Broker is [Scorpio](https://github.com/ScorpioBroker/ScorpioBroker), mainly due to good cloud integration and overall support.
 
 #### Infrastructure requirements
 
-> ex. internal endpoint for the TMForum APIs ; external endpoint for the context broker API ; other endpoints between components ?
+The base memory consumption per deployed pod is listed below but is will increase with the amount of traffic, therefor should only be used as a rough estimate.
 
-#### How to deploy
+| Service          | Memory (Mi) |
+|------------------|-------------|
+| TM-Forum-API Pod | 250         |
+| Scorpio          | 400         |
+| Postgis          | 150         |
+| Redis            | 10          |
+|                  |             |
+|                  |             |
 
-> incl. where to find clean, versioned, infrastructure agnostic HELM charts.
+Apart from the database service, no other service will maintain a own persistence, therefor only for this service a persistent volume claim has to be dimensioned.
 
+#### How to deploy
+The recommended and endorsed way of deployment is via the provided helm charts ( optionally wrapped in ArgoCD Applications).
+
+To deploy a setup, the [umbrella chart](https://helm.sh/docs/howto/charts_tips_and_tricks/#complex-charts-with-many-dependencies) of the access-node can be used as followed:
+
+- create a configuration values file according to the own environment, as described [here](#how-to-configure).
+- add helm chart repository to helm installation
+  ```
+    helm repo add dome-access-node https://dome-marketplace.github.io/access-node
+    helm repo update
+  ```
+  > :bulb: All releases of the Access-Node reside in the helm-repository https://dome-marketplace.github.io/access-node. In addition to that, all Pre-Release versions(build from the Pull Requests) are provided in the pre-repo https://dome-marketplace.github.io/access-node/pre. The pre-repo will be cleaned-up from time to time, in order to keep the index manageable.
+
+- install the components using the prepared configuration
+  ```
+    helm install <RELEASE_NAME> dome-access-node/access-node --namespace <NAME_SPACE> --version <CHART_VERSION> -f values.yaml
+  ```
 #### How to configure
 
-> Recommended configuration for all the sub-components, any considerations. When should I change them?
+The chart is released with a set of [default values](https://github.com/DOME-Marketplace/access-node/blob/main/charts/access-node/values.yaml) which act as a good starting point for an adoption. These values are also documented, enhancing the understanding. Additionally, the [respective charts](https://github.com/FIWARE/helm-charts/tree/main/charts/tm-forum-api) of the components should be consulted.
```

Comment:
Instead of just linking the tmforum-api chart, maybe add a table or list so that we can add the links to the different charts for all components? That means not only tmforum-api, but also the chart repos for scorpio, blockchain-connector (to be added then), maybe even linking the chart repos of the 3rd party components. This way, the integrator has all charts at hand and can look up all config parameters. 
They are also in the snippet below (and can stay there), but this way the reader can directly open them.
What do you think?